Make municipality name optional in the output

jwkaltz opened the pull request:
Fixes #129 , replaces #130 :
Make municipality name optional in the output.
Must be done simultaneously to https://github.com/openoereb/pyramid_oereb/pull/1703

Commit:
Make municipality name optional in the output

jwkaltz commented:
Tested on dummy installation, output when parameter is set to true:
![parameter-set-to-true](https://user-images.githubusercontent.com/9261252/219578377-340598ec-8c57-49b4-9098-fa313e453459.png)
Output when parameter is set to false:
![parameter-set-to-false](https://user-images.githubusercontent.com/9261252/219578401-0176c776-b04e-4490-8cb6-2f148882dee4.png)

michmuel commented:
I cannot completely reproduce your example. If I use the respective pyramid oereb branch setting in pyramid_oereb.yml
```
    # Flag to print or not the municipality name
    # print_municipality_name: true
```
or
```
    # Flag to print or not the municipality name
    print_municipality_name: true
``
no municipality name is printed. Do you know why?

peterschaer commented:
I observe the same behaviour as @michmuel does. I can't get the municipality name to be printed.

jwkaltz commented:
Indeed I forgot something, sorry. I will do a new commit.

Commit:
add parameter for subreports

marionb committed:
Pass parameter PrintMunicipalityName to sub-report
To have the parameter available it needs to be passet down though all sub-reports.

marionb commented:
@michmuel and @peterschaer can you test again it should be working with the last commit.

peterschaer reviewed (approved):
Now it works as expected. Thanks! For me it's ok to merge.

marionb commented:
@michmuel do you want to also review? Or is it OK for you to merge this?

michmuel commented:
@marionb It is good for me as Peter has tested it and approved. Many thanks.
